projects/kit/components/input-phone/input-phone.component.ts

```diff
@@ -151,10 +133,18 @@ export class TuiInputPhoneComponent
         );
     }
 
-    get computedValue(): string {
-        return this.value
-            ? tuiFormatPhone(this.value, this.countryCode, this.phoneMaskAfterCountryCode)
-            : this.search || '';
+    get nativeValue(): string {
+        return this.nativeFocusableElement
+            ? this.nativeFocusableElement.value
+            : maskitoTransform(this.value, this.maskOptions);
+    }
+
+    set nativeValue(value: string) {
+        if (!this.nativeFocusableElement) {
+            return;
```

Comment:
nit: with one line we do not do early returns

Reply:
fixed